README.MD

```diff
@@ -0,0 +1,70 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
+
+
+## Built With
+
+- HTML
+- CSS
+- Microverse linters
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/SekoViper/Hello-Microverse/blob/main/.gitignore.txt)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/SekoViper/Hello-Microverse/blob/main/.gitignore.txt)
+- Twitter: [@twitterhandle](https://twitter.com/SekoViper)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/livingstone-dameh/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] It would be a good thing to provide your visitors to your repository a way to easily navigate to the issues page of this repo. Consider adding this line ```https://github.com/SekoViper/Microverse-Hello/issues```